enigma/src/main/java/org/quiltmc/enigma/api/Enigma.java

```diff
@@ -130,16 +115,13 @@ public EnigmaProject openJar(Path path, ClassProvider libraryClassProvider, Prog
 		ProjectClassProvider projectClassProvider = new ProjectClassProvider(mainProjectProvider, librariesProvider);
 
 		// main index
-		this.index(jarIndex, projectClassProvider, progress, "jar", false, null);
-
-		// TODO make filtering toggleable with arg once JavaClassProvider is used
-		final Predicate<String> mainReferencedPredicate = this.createMainReferencedPredicate(jarIndex, projectClassProvider);
+		this.index(jarIndex, projectClassProvider, progress, "jar", false);
 
 		// lib index
-		this.index(libIndex, projectClassProvider, progress, "jar", true, mainReferencedPredicate);
+		this.index(libIndex, projectClassProvider, progress, "jar", true);
```

Comment:
this should have its own translation key (bad UX to show the same progress bar twice)